### DISCUSSION TITLE
Is there a way to override or unset blurls option?

### DISCUSSION THREAD

**Discussion by topazrn** [hyprwm/Hyprland#4033] [Q&A] [answered]
My config:

~/.config/hypr/hyprland.conf:
```conf
source = ~/.config/hypr/themes/theme.conf
source = ~/.config/hypr/userprefs.conf
```
~/.config/hypr/themes/theme.conf:
```conf
blurls = waybar
```
~/.config/hypr/userprefs.conf:
```conf
blurls = false
```

I checked `hyprctl getoption blurls` to get its default value to unset it but it outputs `no such option`. So I'm asking if anyone know how to override or unset the blurls option.

**Comment by vaxerski** [comment, marked as answer]
switch away from the deprecated `blurls` to a `layerrule` and you will notice in the wiki how to unset it.

**Reply by topazrn** [reply]
Yup that works. Thanks Vaxry.
